components/layout/display_list/webrender_helpers.rs

```diff
@@ -225,6 +225,7 @@ impl WebRenderDisplayItemConverter for DisplayItem {
 
                 builder.push_stacking_context(
                     &webrender_api::LayoutPrimitiveInfo::new(stacking_context.bounds),
+                    Some(scrolling_id),
```

Comment:
It's the only `ClipId` I found in that function. Maybe it should be None ?

Reply:
Yup, should be None for now, to retain existing behavior.